package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "multer-s3",
-  "version": "2.10.0",
+  "version": "3.0.0",
```

Please don't bump the version in this PR, that will be done with `npm version` when we release. This is to ensure proper tags and GitHub Releases...

```suggestion
  "version": "2.10.0",
```